client/state/notices/README.md

```diff
@@ -0,0 +1,53 @@
+Notices
+===========
+
+A module for managing global notices.
+
+## Create a notice
+
+Provides some helper functions to render a notice on the UI. Types of notices are `error`, `success`.
```

Comment:
I'm not sure this text works for "Create a notice". Do we actually need this? Maybe just let the first paragraph stand, and augment with something like "Supports error and success notices."